### DISCUSSION TITLE
Is it possible to have a private repository that can have it so other members can't edit it(only the owner)

### DISCUSSION THREAD

**Discussion by shadow00dev** [community/community#40166] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

basically, as the title says

Is it possible to have a private repository that can have it so other members can't edit it(only the owner)

note: I don't wanna use organizations if possible

**Comment by airtower-luna** [comment]
For a user (non-organization) repository that's not possible. If you add someone as a contributor they get write access.

**Reply by shadow00dev** [reply]
okay, thanks

**Comment by jfullstackdev** [comment]
why would you not want Organizations ? it's a very good feature, you can even organize your repos using Orgs, you being the only member or with others

**Reply by shadow00dev** [reply]
to b me honest  I'm not sure why i don't want to use it

**Comment by jfullstackdev** [comment, marked as answer]
maybe try this 

https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch

then at Settings, 

![image](https://user-images.githubusercontent.com/47092464/204122814-053c94fc-0620-48b5-a2f9-7945caa03e6d.png)

set the Base permission to Read Only

plus you all have the perks of having an Org, like private chatting with members etc.

**Reply by shadow00dev** [reply]
okay thanks
